export/wiki/species/xp.py

```diff
@@ -126,4 +128,21 @@ def convert_level_data(species: PrimalDinoCharacter, dcsc: DinoCharacterStatusCo
 
         result.wildLevelTable = out_level_table
 
+    # Calculate required amount of Mutagen needed to use it.
+    # Formula extracted from PrimalItemConsumable_Mutagen_C's scripts
+    # Vultures cannot use Mutagen as it can't enter their inventories.
+    # IsRobot and IsVehicle banned from Mutagen in v329.51.
+    if species.DinoNameTag[0] != 'Vulture' and not species.bIsRobot[0] and not species.bIsVehicle[0]:
```

Comment:
As discussed, I'd like to get the Vulture tag removed from the code and replaced by a configurable list in overrides. Please create an issue for this work to be done in future.